stacks/analytics-ingest-lambda.yml

```diff
@@ -18,6 +18,14 @@ Parameters:
     Type: String
```

Comment:
Since this file has gotten so…robust, I wouldn't hate some comments/docs at the top to give a brief overview of what's all happening